src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -86,6 +87,14 @@
         DESC_CS2103T_TUTORIAL = new EditModuleClassDescriptorBuilder().withName(VALID_NAME_CS2103T_TUTORIAL).build();
     }
 
+    // lesson-related constants
+
+    public static final String VALID_START_TIME_WED_2_TO_4 = "14:00";
+    public static final String VALID_END_TIME_WED_2_TO_4 = "16:00";
+    public static final Day VALID_DAY_WED_2_TO_4 = Day.WEDNESDAY;
+    public static final int VALID_NUMBER_OF_OCCURRENCES_WED_2_TO_4 = 7;
+    public static final String VALID_VENUE_WED_2_TO_4 = "COM1-B111";
```

Comment:
Perhaps naming the variables as they are is good enough?
```suggestion
    public static final String VALID_TIME_1400 = "14:00";
    public static final String VALID_TIME_1600 = "16:00";
    public static final Day VALID_DAY_WED = Day.WEDNESDAY;
    public static final int VALID_NUMBER_OF_OCCURRENCES = 7;
    public static final String VALID_VENUE = "COM1-B111";
```